### DISCUSSION TITLE
Self nomination for php contributor

### DISCUSSION THREAD

**Discussion by TisVictress** [paketo-buildpacks/feedback#137] [PHP Team]
I've been working on getting php compatible with jammy. I'd love to be a contributor!
https://github.com/paketo-buildpacks/php-dist/commit/b8ea05aa3effb883147593e22083000c029a26ae

**Comment by TisVictress** [comment]
CC @paketo-buildpacks/steering-committee

**Comment by ryanmoran** [comment]
Congrats @TisVictress on your approval to become a PHP Contributor! 🎉
